ironfish/src/syncer.ts

```diff
@@ -365,9 +365,11 @@ export class Syncer {
           break
         }
 
-        peer.sequence = block.header.sequence
-        peer.head = block.header.hash
-        peer.work = block.header.work
+        if (!peer.sequence || block.header.sequence > peer.sequence) {
```

Comment:
should this be above the break?

Reply:
If we do that, we'll update their sequence to be equal to an invalid block, right?

Reply:
hm okay. they did send it, but sure yeah i see your point